AmericanMedicalAssociation/AMA-style-guide
# Ewl 3078 how to styleguide

Ticket: https://issues.ama-assn.org/browse/EWL-3078

## Testing instructions
* review the [`README`](https://github.com/AmericanMedicalAssociation/AMA-style-guide/tree/EWL-3078-how-to-styleguide)
* Observe you understand what the style guide is and how to use it
